### ISSUE TITLE
Disabling default features fails to compile

### ISSUE THREAD

**Issue by mcobzarenco** [Bluefinger/bevy_rand#37]
Disabling default features fails to compile the crate, e.g.
```
cargo new --bin test-bevy-rand
cd test-bevy-rand
cargo add bevy_rand --no-default-features
cargo build
```
fails with
```
   Compiling bevy_rand v0.8.0
error[E0425]: cannot find value `seed` in this scope
   --> /home/marius/.cargo/registry/src/index.crates.io-6f17d22bba15001f/bevy_rand-0.8.0/src/traits.rs:247:23
    |
247 |             getrandom(seed.as_mut()).expect("Unable to source entropy for seeding");
    |                       ^^^^ not found in this scope

For more information about this error, try `rustc --explain E0425`.
error: could not compile `bevy_rand` (lib) due to 1 previous error
```

**Comment by Bluefinger** [comment]
Oops. All I can say about this one :smile:

**Comment by mcobzarenco** [comment]
Thank you! :raised_hands: I see that main tracks bevy main, would it be too much of a pain to have a patch release with this fix to enable using it with bevy `0.15`? Just a thought, you can always just enable all features as a workaround.

**Comment by Bluefinger** [comment]
I think there's some breaking changes in for my next bevy `0.15` release, which I have a branch for where I cherry pick the commits I want in for the current release. I'll probably be doing a new version sooner rather than later, as I am finding more things that are broken by default
